src/content/contributing/index.md

```diff
@@ -1,12 +1,313 @@
 ---
 title: Contributing
 lang: en
+sidebar: true
 ---
 
-# Contributing to ethereum.org
+# Contributing to ethereum.org 🦄
 
-This page is coming soon!
+ethereum.org is an open-source project. So if you want to help improve our portal to Ethereum, here's how you can help out.
```

Comment:
```suggestion
ethereum.org is an open-source project. So if you want to help improve [our portal to Ethereum](https://ethereum.org/en/about/), here's how you can help out.
```

Perhaps a good spot to link to the /about/ page in case they haven't seen it.